src/Database/Attach/File.php

```diff
@@ -119,7 +119,9 @@ public function fromPost($uploadedFile)
             ? $uploadedFile->getPath() . DIRECTORY_SEPARATOR . $uploadedFile->getFileName()
             : $uploadedFile->getRealPath();
 
-        $this->putFile($realPath, $this->disk_name);
+        if (!$this->putFile($realPath, $this->disk_name)) {
+            throw new ApplicationException('The file failed to be stored');
```

Comment:
Is there any more detailed information we can get out of the result of putFile() to include in the error message?

Reply:
@jaxwilko

Reply:
Put file just returns `bool`